### ISSUE TITLE
Some modules does not have nodeSelector/tolerations tunables.

### ISSUE THREAD

**Issue by lazovskiy** [deckhouse/deckhouse#4687]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Version

v1.44

### Expected Behavior

I have a cluster that has master and worker nodes. It has no dedicated system nodes.
This leads to various system components are running on generic nodes.
I want to pin those components to master nodes, as I have plenty of resources. There are several ways to accomplish this:

1. I can put the "system node" label on the master NodeGroup nodeTemplate. In this case, the deckhouse will automatically deploy all its components to that nodes. But as the master nodes have taints, this will fail because of a lack of tolerations on deployed resources. So I have to configure tolerations on each affected component explicitly.
2. I can configure every module with nodeSelector+tolerations

Removing taints from the master nodes is not an option, as it may lead to unsolicited payload running on masters.

Both methods require tolerations or/and nodeSelector options to be available in every single module configuration. Still, as for now, this requirement does not meet, at least in the following modules:
* snapshot-controller
* descheduler
* linstor
* admission-policy-engine

In short, every module that deploys pods should have configuration options for nodeSelector and tolerations.

### Actual Behavior

Some modules do not have configuration or lack the nodeSelector/tolerations options.

**Comment by konstantin-axenov** [comment]
Only snapshot-controller is left.
